packages/hardhat-chai-matchers/test/contracts/Events.ts

```diff
@@ -34,7 +34,7 @@ pragma solidity ^0.6.0;
           emit Two(2, "Two");
       }
 
-      function _emitInternal() internal {
```

Comment:
Not sure why this was necessary, or how it was working in the Waffle codebase. Maybe because they're using a newer version of ethers (5.6.1) than us (5.0.0)?